docs/source/app-dev/bindings-x-lang/index.rst

```diff
@@ -94,4 +94,13 @@ Links
 
 - gRPC documentation: https://grpc.io/docs/
 
-- DAML Ledger API gRPC protobuf definitions: https://github.com/digital-asset/daml/tree/master/ledger-api/grpc-definitions
+- DAML Ledger API gRPC protobuf definitions
+    - current master: https://github.com/digital-asset/daml/tree/master/ledger-api/grpc-definitions
+    - for specific versions: https://github.com/digital-asset/daml/releases
+
+- External required protobuf definitions:
+    - https://raw.githubusercontent.com/grpc/grpc/v1.18.0/src/proto/grpc/status/status.proto
+    - https://raw.githubusercontent.com/grpc/grpc/v1.18.0/src/proto/grpc/health/v1/health.proto
+
+- External optional protobuf definitions:
+    - https://raw.githubusercontent.com/grpc/grpc/v1.18.0/src/proto/grpc/reflection/v1alpha/reflection.proto
```

Comment:
Hm, we never refer to this in any protobuf file, but it's only used for the reflection service, so not really needed for building bindings. WDYT?

Reply:
Don't you need it to make something that interacts with the reflection service? I added it to the optional category.

Reply:
Yeah, thinking better about it, I think it makes sense to just drop this.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/5664/commits/9e09c02dd823490a7f35019b1d348ae226bfe395

Reply:
sure, but we pretty much only added the reflection service so that `grpcurl` works nicely, iirc.